VideoApp/VideoApp/TwilioVideo/Room/RoomManager.swift

```diff
@@ -35,9 +35,11 @@ class RoomManager: NSObject, ObservableObject {
     @Published var room: Room? // Only exposed for stats
     private(set) var localParticipant: LocalParticipantManager!
     private(set) var remoteParticipants: [RemoteParticipantManager] = []
+    private var captionsManager: CaptionsManager!
```

Comment:
I considered inverting this dependency so that `RoomManager` doesn't own so many things. But I think that would increase abstractions and probably not be as easy to follow. So just handled this similar to `localParticipant`.